.github/workflows/run-tests.yml

```diff
@@ -309,12 +377,12 @@ jobs:
             ./liquibase-integration-tests/target/jacoco.exec
 
   sonar:
-    needs: [ build_tests, integration-test ]
+    needs: [ setup, build_tests, integration-test ]
     uses: liquibase/build-logic/.github/workflows/sonar-test-scan.yml@v0.7.8
     with:
       thisBranchName: ${{ needs.setup.outputs.thisBranchName }}
-      thisSha: ${{ needs.setup.outputs.thisSha }}
-      liquibaseBranchName: ${{ needs.setup.outputs.liquibaseBranchName }}
+      thisSha: ${{ needs.setup.outputs.latestMergeSha }}
```

Comment:
Are those changes expected?

Reply:
the values are not getting populated here : https://github.com/liquibase/liquibase/actions/runs/9566205290/job/26374745886#step:1:44. But I think I'll delete it for this PR.